### DISCUSSION TITLE
yolo object id

### DISCUSSION THREAD

**Discussion by mickycooky** [ultralytics/ultralytics#15811] [Q&A]
i have question.
if the same person repeat appear don't appear in video, each person is assigned same ID or another ID
sorry my english is not good

**Comment by glenn-jocher** [comment]
@mickycooky if the same person reappears in the video after disappearing, they will likely be assigned a new ID. Tracking IDs are typically reset when the object is lost from view.

**Reply by mickycooky** [reply]
thank you answer. 
i have next one, if the perfectly resemble persons reppear in the 1 frame, they are assigned same ID or another ID?

**Reply by glenn-jocher** [reply]
@mickycooky if two perfectly resembling persons appear in the same frame, they will be assigned different IDs. Each detected object in a frame is given a unique ID.
